src/www/ui/api/documentation/openapi.yaml

```
@@ -757,6 +757,115 @@ paths:
                 $ref: '#/components/schemas/User'
         default:
             $ref: '#/components/responses/defaultResponse'
+    put:
+      operationId: modifyUserById
+      tags:
+        - User
+        - Admin
+      summary: Edit user details by id
+      description: >
+        Edit user details by id
+      requestBody:
+        required: true
+        content:
+          application/x-www-form-urlencoded:
+            schema:
+              type: object
+              properties:
```

Please reuse existing data models instead of creating new ones to keep consistent models.

https://github.com/fossology/fossology/blob/5eaca363ca51c212038939acc98c24672b6392e0/src/www/ui/api/documentation/openapi.yaml#L1669

```suggestion
            schema:
              $ref: '#/components/schemas/User'
```

---

I tried reusing that, but there are couple of new parameters that needs to be added  without which the API wont work.

---

[Update]: I learnt how we can add more parameters to an existing schema using `allOf`. Will do the required changes.